krun/platform.py

```diff
@@ -761,23 +762,61 @@ def find_temperature_sensors(self):
         The sensors move between reboots, so we have to roll our own sensor
         identifier. We are not fussy, and use as many sensors as we can find."""
 
+        ignore_unnamed_chips = False
```

Comment:
I don't understand why this variable is present a) because I think it's meant to be set to `True` b) I don't see why anyone would want it to be set to `False`. Am I missing something?

Reply:
It is set to true on line 793 and checked on 807, so I think this is OK?

Reply:
OK, so maybe the variable is badly name? Should it be something like `found_unnamed_chips'?

Reply:
We ignore un-named chips if we find more than one un-named chip.

Reply:
OK, so I think this variable would then be better as `found_unnamed_chips` (or similar; shorter might be better!) then.